packages/components/src/components/modal/modal.scss

```diff
@@ -23,6 +19,35 @@
         max-height: calc(100vh - 48px - 36px);
         max-width: 90vw;
```

Comment:
Can u include these property rules to within the desktop mixin as well? Since these max values are tailored mainly for desktop.

Reply:
Sure.